extensions/amp-story-interactive/0.1/utils.js

```diff
@@ -47,3 +47,38 @@ export const deduplicateInteractiveIds = (doc) => {
     }
   }
 };
+
+/**
+ * Reorders options data to account for scrambled or incomplete data.
+ *
+ * @param {!Array<!InteractiveOptionType>} optionsData
+ * @param {!Array<!Element>} optionElements
+ * @return {!Array<!InteractiveOptionType>}
+ */
+export const orderData = (optionsData, optionElements) => {
```

Comment:
If we only use the length of optionElements, we could just pass in the number instead of the whole array of elements